cmd/command.go

```diff
@@ -594,6 +595,34 @@ func hIncrBy(cli *Client, args [][]byte) (interface{}, error) {
 	return cli.db.HIncrBy(key, field, incrInt64Val)
 }
 
+func hRandField(cli *Client, args [][]byte) (interface{}, error) {
+	if len(args) < 1 || 3 < len(args) {
+		return nil, newWrongNumOfArgsError("hrandfield")
+	}
+	if len(args) == 1 {
```

Comment:
maybe this is more clear?
```
count, withValues := 1, false
if len(args) == 3 {
    count = xxx
    withValues = xxx
}
return cli.db.HRandField
```

Reply:
I would like to do this as well, but the specification of the return value of redis is different for 1 argument and for 2 or more arguments.

https://redis.io/commands/hrandfield/#return

> Bulk string reply: without the additional count argument, the command returns a Bulk Reply with the randomly selected field, or nil when key does not exist.
> 
> Array reply: when the additional count argument is passed, the command returns an array of fields, or an empty array when key does not exist. If the WITHVALUES modifier is used, the reply is a list fields and their values from the hash.

Therefore, at least the implementation part of the server commands was matched to redis.
On the other hand, the rosedb side returns a [][]byte in all cases.
This is because the redis behavior seems a bit strange.